app/controllers/api/connect/v3/systems/products_controller.rb

```diff
@@ -108,7 +108,7 @@ def create_product_activation
 
     # in BYOS mode, we rely on the activation being performed in
     # `engines/scc_proxy/lib/scc_proxy/engine.rb` and don't need further checks here
-    return activation if @system.proxy_byos
+    return activation if @system.byos?
```

Comment:
Where is the method `System#byos?` defined?

Reply:
in the [language](https://api.rubyonrails.org/v5.1/classes/ActiveRecord/Enum.html) ?

Reply:
#TIL ;-)